com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -470,10 +616,7 @@ private void SendAllParamsAndState()
                 {
                     Send = new ClientRpcSendParams
                     {
-                        TargetClientIds = NetworkManager.ConnectedClientsList
-                            .Where(c => c.ClientId != NetworkManager.ServerClientId)
-                            .Select(c => c.ClientId)
-                            .ToArray()
+                        TargetClientIds = ServerToClientMessagingTargetClientIds
```

Comment:
So the intention with the client / server separation was to have the library be purely server authoritative; but to enable users that might want a ClientNetworkTransform or ClientNetworkAnimator in this case we have those implementations in Samples, just as you have done.  Thing is, this NetworkAnimator itself is has the client auth code in it (line 627), and then we are just activating that code with the ClientNetworkAnimator "CanCommitToAnimator" toggle.  It would be more aligned with what we're trying to do if instead we could do something like override SendAllParamsAndState in the CNA sample.  It probably seems like splitting hairs, but I'd like to go more that direction